qiskit_experiments/curve_analysis/standard_analysis/resonance.py

```diff
@@ -21,50 +21,49 @@
 
 
 class ResonanceAnalysis(curve.CurveAnalysis):
-    r"""A class to analyze a resonance, typically seen as a peak.
+    r"""A class to analyze a resonance, typically seen as a Lorentzian peak.
```

Comment:
The expression is given below but should we clarify somewhere in here that it is the square root of the standard lorentzian function that is used here?